collect_app/src/main/java/org/odk/collect/android/formentry/ODKView.java

```diff
@@ -134,6 +135,9 @@
     @Inject
     StringRequester stringRequester;
 
+    @Inject
+    MediaUtils mediaUtils;
```

Comment:
Again, I think we could avoid Dagger here and just use the constructor.

Reply:
Fixed.